yolo_path: /home/hy/wust_radar/src/wust_radar_detect/model/TensorRT/yolo1280.engine
armor_path: /home/hy/wust_radar/src/wust_radar_detect/model/TensorRT/armor_yolo192.engine
classify_path: /home/hy/wust_radar/src/wust_radar_detect/model/TensorRT/classify.engine
